src/todo-func.js

```diff
@@ -0,0 +1,113 @@
+export const getTodos = () => JSON.parse(localStorage.getItem('todos')) || [];
+export const refresh = () => {
+  renderTodos();// eslint-disable-line
```

Comment:
- [ ]  Hi, your App is working fine but it is prohibited to disable the linter. Can you fix it manually? This is because a [linter](https://github.com/microverseinc/linters-config) is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.

Reply:
:red_circle: [REQUIRED] I'm writing to follow up on the previous reviewer's comment about disabling the linter. As they mentioned, it's prohibited to disable linters in our code reviews. This is because linters are important tools that help to identify potential errors and bugs in our code. By disabling the linter, you're preventing us from catching these potential issues, which could lead to problems down the road.

I understand that you may have had a reason for disabling the linter, but I would ask that you please fix this manually.

**I'm confident that you can find a way to fix this issue without compromising the quality of your code**. :hugs: